src/mahalanobis.jl

```diff
@@ -1,11 +1,65 @@
 # Mahalanobis distances
 
+"""
+    Mahalanobis(Q, skipchecks = false) <: Metric
```

Comment:
I suspect we don't use spaces around `=` in general (but maybe worth checking). Also needs a `;`. Same below.
```suggestion
    Mahalanobis(Q; skipchecks=false) <: Metric
```